app/controllers/serviceProviderButtonBar.js

```diff
@@ -0,0 +1,55 @@
+var args = arguments[0] || {};
+var countySelector = require("countySelectorUtils");
+
+var providerListObject;
+
+exports.setProviderListObject = function(listObject){
+	providerListObject = listObject;
+};
+
+/**
+ * Function that either displays list of service providers or map of
+ * service providers based on user preference. 
+ */
+function toggleMapListView(){
+	var mapModule = providerListObject.getMapViews().mapModule;
+	var map = providerListObject.getMapViews().map;
+	var listView = providerListObject.getListView();
+	
+	if(mapModule.visible){
+		setMapVisibility(false);
+	}
+	else{
+		setMapVisibility(true);
+		map.setRegion({latitude:39.719704, longitude:-84.219832, latitudeDelta:0.2, longitudeDelta:0.2});
```

Comment:
Getting the user's actual location is probably better than using Dayton by default.